tests/python/gaia-ui-tests/gaiatest/apps/lockscreen/app.py

```diff
@@ -29,6 +29,14 @@ class LockScreen(Base):
 
     _time_locator = (By.ID, 'lockscreen-clock-time')
 
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+        # This is to prevent the screen from turning off while on lockscreen, see bug 1169332
+        self.marionette.switch_to_frame()
+        self.marionette.execute_script("""
+            window.wrappedJSObject.ScreenManager.LOCKING_TIMEOUT = 9999;
```

Comment:
That might probably be a better idea, actually. How I've done this, doesn't work for instance with test_lockscreen_notification.py. Because once the lockscreen is shown, the existing screen timeout value is used and not the changed one.

I don't think we have tests that rely on the lockscreen timing out, afaik.